hub/powertoys/run.md

```diff
@@ -23,92 +23,86 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
-| `:` | Registry keys only | `: hkcu` to search for the 'HKEY_CURRENT_USER' registry key. | yes |
-| `!` | Windows services only | `! alg` to search for the 'Application Layer Gateway' service to be started or stopped. | yes |
-| `>` | Shell command only | `> ping localhost` to do a ping query. | yes |
-| `(` | Time and date only | `( time and date` shows the current time and date in different formats.<br />`( calendar week::04/01/2022` shows the calendar week for the date '04/01/2022'. | yes |
-| `&` | Time zones only | `& Newfoundland` shows the current time in the time zone of Newfoundland.| yes |
-| `%%` | Unit converter only | `%% 10 ft in m` to calculate the number of meters in 10 feet. | yes |
-| `//` | URIs only | `//` to launch your default browser.<br />`// docs.microsoft.com` to have your default browser go to https://docs.microsoft.com.<br />`mailto:` and `ms-settings:` links are also supported. | yes |
-| `{` | Visual Studio Code previously opened workspaces, remote machines (SSH or Codespaces) and containers. | `{ powertoys` to search for workspaces that contain 'powertoys' in their paths. | **no** |
-| `??` | Web search only | `??` to launch your default browser's search page.<br />`?? What is the answer to life` to search with your default browser's search engine. | yes |
-| `$` | Windows settings only | `$ Add/Remove Programs` to launch the Windows settings menu for managing installed programs.<br />`$ Device:` to list all settings with 'device' in their area/category name.<br />`$ control>system>admin` shows all settings of the path 'Control Panel > System and Security > Administrative Tools'. | yes |
-| `_` | Windows Terminal profiles only | `_ powershell` to list all profiles that contains 'powershell' in their name. | **no** |
-| `<` | Open windows only | `< outlook` to find all open windows that contain 'outlook' in their name or the name of their process. | yes |
-
-
-
-## Plugins
-
-PowerToys Run uses a plugin system to provide different types of results.
-
-###  Plugin manager
-
-The PowerToys Run settings menu includes a plugin manager that allows you to enable/disable the various available plugins. By selecting and expanding the sections, you can customize the direct activation commands used by each plugin. In addition, you can select whether a plugin appears in global results, as well as set additional plugin options where available.
-
-![PowerToys Run Plugin Manager](../images/pt-run-plugin-manager.png)
-
+| Plug-in                                                                              | Direct activation command | Example                                                                                                                                                                                                                                                                                              |
```

Comment:
Do we mention the plugin or what you can do (`Window Walker` vs `Open windows`)? Should be aligned.

Reply:
This was written before me, but I can make it fit "plugin" as in Settings window.